website/content/docs/platform/k8s/helm/configuration.mdx

```diff
@@ -648,6 +648,14 @@ and consider if they're appropriate for your deployment.
 
     - `enabled` (`boolean: true`) - When set to `true`, a Kubernetes service will be created for Vault.
 
+    - `active` - Values that apply only to the vault-active service.
+
+      `enabled` (`boolean: true`) - When set to `true`, the vault-active Kubernetes service will be created for Vault, selecting pods which label themselves as the cluster leader with `vault-active: "true"`.
```

Comment:
It looks like this should be a separate bullet under `active`?

```suggestion
      - `enabled` (`boolean: true`) - When set to `true`, the vault-active Kubernetes service will be created for Vault, selecting pods which label themselves as the cluster leader with `vault-active: "true"`.
```

Reply:
Oops, you're right, thanks for catching that